### ISSUE TITLE
U+1133C GRANTHA SIGN NUKTA blocks Tamil ligatures

### ISSUE THREAD

**Issue by dscorbett** [notofonts/tamil#2]
### Font
NotoSerifTamil-Regular.otf
NotoSerifTamilSlanted-Regular.otf

### Where the font came from, and when
Site: https://github.com/googlefonts/noto-fonts/blob/115d38430d957d38307457c036302b7bdbe0bbc4/phaseIII_only/unhinted/otf/NotoSerifTamil/NotoSerifTamil-Regular.otf
Site: https://github.com/googlefonts/noto-fonts/blob/115d38430d957d38307457c036302b7bdbe0bbc4/phaseIII_only/unhinted/otf/NotoSerifTamilSlanted/NotoSerifTamilSlanted-Regular.otf
Date: 2020-04-08

### Font version
Version 2.000

### Issue
U+1133C GRANTHA SIGN NUKTA blocks ligatures. Noto Sans Tamil gets it right.

### Character data
க𑌼ுகு
U+0B95 TAMIL LETTER KA
U+1133C GRANTHA SIGN NUKTA
U+0BC1 TAMIL VOWEL SIGN U
U+0B95 TAMIL LETTER KA
U+0BC1 TAMIL VOWEL SIGN U

### Screenshot
![க𑌼ுகு](https://user-images.githubusercontent.com/1124347/78794583-f49ea080-7981-11ea-9e87-391c3dc5ed5d.png)
![க𑌼ுகு](https://user-images.githubusercontent.com/1124347/78794584-f49ea080-7981-11ea-938d-b59e7ff10d82.png)

**Comment by simoncozens** [comment]
@virtualvinodh, could you give an opinion on where Grantha nuktas should go on Tamil glyphs? (Or if it matters?)

![shape](https://user-images.githubusercontent.com/106728/225852337-0cebd2e5-dbea-406f-b108-5c24f78d948f.png)
![shape](https://user-images.githubusercontent.com/106728/225852406-3e13cc39-bdd8-4c78-ac36-df165540e6c2.png)

**Comment by virtualvinodh** [comment]
Personally, I have never used Nuktas with Tamil letters.

If I were to use one, I would go for the second image. Looks more natural. 

V

**Comment by simoncozens** [comment]
Thanks!

**Comment by jamadagni** [comment]
I agree with @virtualvinodh that it would be simpler, cleaner and more consistent to keep the nukta below the consonant+vowel ligature, but it is possible that it was done due to line spacing considerations? Normally Tamil is a vertically very compact but horizontally more space-using script, so please check that before making any changes.

**Comment by jamadagni** [comment]
FWIW I should note that the OP shows the consonant+vowel ligature broken by the nukta. Please ensure that it is not broken so. What is shown by @simoncozens in the last image above is correct.
